src/providers/anime/animesuge/create-animesuge.ts

```
@@ -0,0 +1,327 @@
+
+  const fetchEpisodeServers = async (episodeId: string, subOrDub: SubOrDub): Promise<IEpisodeServer[]> => {
+    try {
+      if (episodeId.includes('$episode$')) episodeId = episodeId.split('$episode$')[1]!;
+      const response = await fetch(`${config.baseUrl}/ajax/v2/episode/servers?episodeId=${episodeId}`);
+      const data = await response.json();
+      console.log(data);
+      const $ = load(data.html);
+      const scrapedServers: any[] = [];
+      let selector;
+      try {
+        selector = `.ps_-block.ps_-block-sub.servers-${false ? 'raw' : subOrDub} > .ps__-list .server-item`;
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)